### PR TITLE
added rain clouds and smog to landing page (hidden with js)

### PR DESCRIPTION
**Pull request by aidanman29**
Added rain clouds and smog to landing page, although, they are currently hidden with a js statement. Feel free to comment out the statement in order to see the smog or rain cloud images display.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -8,9 +8,50 @@
 }
 
 body{
-  background-image: url(./images/background-image-main.gif);
+  background-image: url(../images/background-image-main.gif);
```
Comment (jypara-git) [inline assets/css/style.css:11]: The actual image needs to be pushed into assets/images folder

**Review by jypara-git — CHANGES_REQUESTED** [review]
images

**Comment by aidanman29** [comment]
did that work? If it didn't im gonna need to zoom with someone to troubleshoot real quick. I think it's just an issue with my git push/pulling stuff.

**Review by jypara-git — APPROVED** [review]
Looks good. Lets try this out

**Pull request merged**